cmd/courier/watch.go

```diff
@@ -0,0 +1,82 @@
+package courier
```

Comment:
Can you please add a test to see if this works (e.g. health port) as expected?

Reply:
Yes, much needed. Where would the best place be to add it?

Reply:
Two options: you extract as much from the cobra handler as possible (e.g. as a function) and then set it up, or you use the cobra test framework to run the command.

I would opt for the first option. Regarding courier tests and the SMTP integration, check out:

- https://github.com/ory/kratos/blob/master/courier/courier_test.go#L41

Cobra commands can be controlled from tests using:

```
import "github.com/ory/x/cmdx"

// ...
ce := &cmdx.CommandExecuter{New: func() *cobra.Command {
		return cmd.RootCmd // might need to be exported in kratos
	}, Ctx: ctx}

ce.ExecNoErr(t, "your", "command")
```

```
```

Reply:
Added some tests in `cmd/courier/watch_test.go`, let me know what you think!